product_docs/docs/pem/8/tuning_performance/performance_diagnostic.mdx

```diff
@@ -11,7 +11,9 @@ redirects:
 The Performance Diagnostic dashboard analyzes the database performance for Postgres instances by monitoring the wait events. To display the diagnostic graphs, PEM uses the data collected by the EDB Wait States module.
 
 !!! Note
-    For PostgreSQL databases, Performance Diagnostics is supported only for versions 10, 11, and 12 that are installed on the supported CentOS or RHEL platforms.
```

Comment:
Is it Performance Diagnostic or Diagnostics? The preceding paragraph. says Diagnostic. If you're using it as a generic term, you could lowercase it,  in which case you need a plural verb for agreement: 
performance diagnostics are supported

Reply:
Diagnostic